### ISSUE TITLE
Print Trades too Logbook

### ISSUE THREAD

**Issue by Misunderstood-Wookiee** [Misunderstood-Wookiee/Mules-and-Warehosues-Extended#4]
Print Trade Completion results too player logbook

**Comment by bhayden53** [comment]
example for travel mule in https://github.com/Misunderstood-Wookiee/Mules-and-Warehosues-Extended/commit/a587a705678ca3d9b8c04ec09b02b45ad0cd1b40

**Comment by Misunderstood-Wookiee** [comment]
Noticed that one.. will have to get an output like this for station mule. 
Unsure if we should default log tasks for distribution mule and supply mule I think I may just make those two a user-selectable choice if they want that mule to log or not, same with the travel one after I implement logging that works for the rest will tie the function into a selectable option to avoid un-necessary logging of mules you don't care about.

**Comment by bhayden53** [comment]
I think it might be nice to have the option for the supply and distrib mules to log to the player logbook, since you can easily filter and check on supply lines that way.

**Comment by Misunderstood-Wookiee** [comment]
> I think it might be nice to have the option for the supply and distrib mules to log to the player logbook, since you can easily filter and check on supply lines that way.

Did we get around to doing this?

**Comment by bhayden53** [comment]
This isn't finished yet, it's been blocked by the more pressing issues. I want to make a pass through every mule and put standard messages in there for all of the trades.

**Comment by Misunderstood-Wookiee** [comment]
If we could colour the messages that would be fancy. 
Beyond scope tho.

**Comment by bhayden53** [comment]
In general, logging is much improved. In the interest of cleaning up our issues list, I'm going to close this. Any specific logging requests  from here on can get it's own issue.
